webcompat/static/js/lib/dashboard.js

```diff
@@ -47,6 +47,27 @@ const filteringSort = issuesNumber => {
   });
 };
 
+/* viewMode */
+const viewMode = () => {
+  const view = document.getElementById("js-Filter-view").value;
+  const Triages = document.getElementById("js-Triages");
+  const currentView = document.getElementById("js-Triages").dataset["view"];
+  if (currentView === view) {
+    return;
+  }
+  /* set value */
+  Triages.dataset["view"] = view;
+  localStorage.setItem("DashboardTriageView", view);
+};
+
+const setView = (view = null) => {
+  if (null == view) {
```

Comment:
[yoda conditionals](https://en.wikipedia.org/wiki/Yoda_conditions) always seem weird to me. :p

Reply:
Ah ah 🙃